src/MarkdownTextInput.web.tsx

```diff
@@ -337,14 +338,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
         }
         const changedText = e.target.innerText;
 
-        if (compositionRef.current) {
-          updateTextColor(divRef.current, changedText);
+        const nativeEvent = e.nativeEvent as MarkdownNativeEvent;
+        if (compositionRef.current && (nativeEvent.inputType !== 'insertCompositionText' || nativeEvent.data !== '*')) {
+          updateTextColor(divRef.current, e.target.innerText);
```

Comment:
You have removed the change introduced by this [PR](https://github.com/Expensify/react-native-live-markdown/pull/342/files#diff-ec487e4a4ccb9c4ffd7ead6ff0a585595aa8d9e9b7d5167d0a2444d70d2001e1R341).
```suggestion
          updateTextColor(divRef.current, changedText);
```